core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/MethodConfigResolverSpec.scala

```diff
@@ -306,7 +306,7 @@ class MethodConfigResolverSpec extends WordSpecLike with Matchers with TestDrive
       // failure cases
       assertResult(true, "Missing values should return an error") {
         runAndWait(testResolveInputs(context, configGood, sampleMissingValue, littleWdl, this)).get("sampleMissingValue").get match {
-          case List(SubmissionValidationValue(None, Some(_), intArg)) if intArg == intArgNameWithWfName => true
+          case Seq(SubmissionValidationValue(None, Some(_), intArg)) if intArg == intArgNameWithWfName => true
```

Comment:
`testResolveInputs` returns a `Map[String, Seq[SubmissionValidationValue]]`, yet we were `match`-ing on `List`, not `Seq`. I'm not exactly sure why this worked with Slick 3.3.0 and failed with Slick 3.3.2, but it seems like an appropriate test-code change.